includes/emails/html-mechant-notification.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
The file name is off, missing an `r` in `merchant`

Reply:
Nice catch! This was fixed in the commit d6c6277